atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -1,10 +1,10 @@
 type: scanner
 scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
-default_scan: redhat-release
+default_scan: pipeline-scanner
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
     args: ['python', 'scanner.py', 'release'],
-    description: "Find Red Hat release"
```

Comment:
@dharmit : that will require changing the scanner itself and worker a bit, but meanwhile we want to keep the wrong description for this scanner?